google/resource_runtimeconfig_config.go

```diff
@@ -0,0 +1,148 @@
+package google
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/runtimeconfig/v1beta1"
+)
+
+var runtimeConfigFullName *regexp.Regexp
+
+func init() {
+	runtimeConfigFullName = regexp.MustCompile("^projects/([^/]+)/configs/(.+)$")
+}
+
+func resourceRuntimeconfigConfig() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceRuntimeconfigConfigCreate,
+		Read:   resourceRuntimeconfigConfigRead,
+		Update: resourceRuntimeconfigConfigUpdate,
+		Delete: resourceRuntimeconfigConfigDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateGCPName,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceRuntimeconfigConfigCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	name := d.Get("name").(string)
+	fullName := resourceRuntimeconfigFullName(project, name)
+	runtimeConfig := runtimeconfig.RuntimeConfig{
+		Name: fullName,
+	}
+
+	if val, ok := d.GetOk("description"); ok {
```

Comment:
I think you can simply this by removing the if-check and keeping only:

```go
runtimeConfig.Description = d.Get("description").(string)
```

If description is not set, `d.Get("description").(string)` will return the default value `""` (empty string).